### DISCUSSION TITLE
Users in the app but  'last_active' not working

### DISCUSSION THREAD

**Discussion by jnar** [codeigniter4/shield#646] [Q&A] [answered]
Hello, Im working with  shield for user management, I want to show 'last_active' but is always null.

In auth config I enable:

`public bool $recordActiveDate = true;`

but still nothing, is there anything Im missing?
![image](https://user-images.githubusercontent.com/5036078/220474747-a5016758-789f-4af8-9ffb-2a648507c088.png)

**Comment by datamweb** [comment, marked as answer]
Hello, welcome to Shield.

I guess you have not done the necessary settings to **protect the routes**.
The function of `last_active` depends on the setting of **session/tokens/chain filter** even when `public bool $recordActiveDate = true;` is set.
So I expect you to do the following explanation and check again.

https://codeigniter4.github.io/shield/references/controller_filters/#protect-all-pages

**Reply by jnar** [reply]
Thank you that solved the issue
![image](https://user-images.githubusercontent.com/5036078/220660449-4e9e35cf-8cd3-4aa4-8f30-8ba34bc43dc9.png)
